.serena/memories/codebase_structure.md

```
@@ -0,0 +1,88 @@
+# huerd Codebase Structure
+
+## Directory Layout
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)